src/content/release-notes.mdx

```diff
@@ -5,6 +5,32 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.17.0
+
+23 January 2024
+
+This versions is compatible with Kubernetes versions 1.25 to 1.28.
+
+### New Features
+
+- Enable search in namespace resources UI  <!-- 7577 -->
+- Enable admins to [transfer namespace ownership](core/components/namespaces.mdx#transfer-namespace-ownership)  <!-- 7593 -->
+- Add [Pod metrics to Okteto Insights](admin/okteto-insights.mdx#pod-metrics)  <!-- 7594 -->
+
+### Improvements
+
+- Hoist Personal Access Tokens menu from Setup to Settings  <!-- 7574 -->
+- Rename Launch → Deploy in the Okteto UI  <!-- 7576 -->
+- Rename Secrets -> Variables under both Settings and Admin sections. This rename is for consistency in the UI and delivers no change in functionality   <!-- 7581 -->
```

Comment:
Nit: It seems arrows are different in these 2 lines. The first one seems an icon, could we use the same in both cases?